traveler-swift-core/TravelerKit/Models/AnyItem.swift

```diff
@@ -10,27 +10,44 @@ import Foundation
 
 /// An item of any kind
 struct AnyItem: Decodable {
-    let bookingItem: BookingItem?
-    let parkingItem: ParkingItem?
+    var bookingItem: BookingItem?
+    var parkingItem: ParkingItem?
     let type: ProductType
 
     enum CodingKeys: String, CodingKey {
-        case purchaseStrategy = "purchaseStrategy"
+        case id = "id"
+        case title = "title"
+        case subTitle = "subTitle"
+        case thumbnail = "thumbnail"
+        case price = "priceStartingAt"
```

Comment:
If we need to denote a range it will most likely be in a different field for a different case, for a different attribute, otherwise we will be causing breaking changes to the people implementing the SDK. Changing the individual case right now (i.e. renaming it) won't break the code but it also won't add much value.